platforms/nuttx/cmake/px4_impl_os.cmake

```diff
@@ -67,6 +67,8 @@ function(px4_os_add_flags)
 	# prevent using the toolchain's std c++ library
 	add_compile_options($<$<COMPILE_LANGUAGE:CXX>:-nostdinc++>)
 
+	add_compile_options($<$<COMPILE_LANGUAGE:CXX>:-fno-sized-deallocation>)
```

Comment:
Why do we need that here as well?

Reply:
The flags set by `add_compile_options` aren't picked up by the NuttX build.